acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
```

Comment:
This sounds a bit scary. Even simplest changes to `setup.py` tend to cause problems as we've seen with #1914. I hope you're ready to receive a bunch of bug reports from affected users. Or maybe it's better to perform such audit? In fact, you may want to have an automatic script that does it (before merging this PR), as any dep version bump might cause troubles.